meilisearch-release-changelog/v1.1.0.md

```diff
@@ -0,0 +1,178 @@
+
+⚠️ The experimental feature API can break between 2 versions of Meilisearch. Be careful if using it in production.
+
+Done by @irevoire, @dureuill, and @james-2001 in #3496, #3524, #3538 and #3543.
+
+## Improve the language support
+
+* Enhance Greek support by normalizing diacritics and final sigma
+* Enhance Arabic support by ignoring Tatweel
+* Enhance language detection: the language is now detected when indexing to prevent from wrong language detections during the search
+
+Done in #3347 and on the [Charabia side](https://github.com/meilisearch/charabia/releases/tag/v0.7.1) by @choznerol, @cymruu, @james-2001 and @ManyTheFish.
+
+## Other improvements
+
+* Allow using wildcards (`*`) at the end of index names when creating API keys or tenant tokens (#3174) @Kerollmops & @akhildevelops
+* Customize the CSV delimiter (default: `,`) by using the `csvDelimiter` parameter during the document addition and update (#3505, #3534) @MixusMinimax and @irevoire
+* Improve error messages: introduction of "did you mean ...?" suggestions when doing typos on a Meilisearch parameter (#3492) @irevoire
+* Batch together addition and deletion tasks to speed up indextation. You don't have to do anything to activate it, just enjoy! (#3470) @irevoire
+* Accept `null` as a valid `_geo` field when importing or updating documents (#3515) @irevoire
+* Reduce crate size (from ~200MB to ~50MB) by compressing dictionaries (#3347) @ManyTheFish
+<!-- The following line should NOT be put in RC0 release changelogs because not present yet -->
+* Cache the result of the indexes stats (#3541) @irevoire
+
+# Fixes 🐞
+
+* Enhance facet string normalization: `sortBy` is now correct for accented characters (#3372) @ManyTheFish
+* Fix web-interface issues: bump version of mini-dashboard to [v0.2.6](https://github.com/meilisearch/mini-dashboard/releases/tag/v0.2.6) (#3514) @bidoubiwa
+* Unify "Bad latitude" and "Bad longitude" errors (#3479) @cymruu
+* Prevent from "Cannot allocate memory" errors when creating a lot of indexes (#3331) @dureuill
+* Fix Japanese support (see [the related issue](https://github.com/meilisearch/meilisearch/issues/2403)) thanks to the new language detection (#3347) @ManyTheFish
+
+# Misc
+
+The following changes do not impact user experience.
+
+* GitHub CIs & test suite
+  * Create a preview environment for every PR using Uffizzi (#3040 and #3482) @waveywaves
+  * Add tests with rust nightly in CI (#3495) @ztkmkoo
+  * Add `--all` to test CI (#3403) @curquiza
+  * Remove cache from test CI to prevent Windows CI failures (#3422) @curquiza
+  * Pin Rust version in Clippy job (#3451) @curquiza
+  * Update comments in version bump CI (#3519) @curquiza
+  * Fix cargo flaky (#3348) @irevoire
+  * Make clippy happy for Rust 1.67, allow `uninlined_format_args` (#3434) @dureuill
+* Documentation
```

Comment:
Can we use a different name for this to avoid users reading it as a changelog for the documentation website?